README.md

```diff
@@ -461,6 +461,73 @@ TL;DR for a simple backup of three PostgreSQL databases in the same project name
 NOTE the `BACKUP_VOLUME_NAME=bk-b7cg3n-deploy-yxq6rf8z23pu` is from Step 2 above; when using the GUI there is no option to set the PVC name.
 </details>
 
+<details><summary>Concrete example of a MongoDB deployment</summary>
+
+
+TL;DR: A simple backup of a single MongoDB database with backup validation.
+
+1. Decide on amount of backup storage required (5Gi is currently the maximum)
```

Comment:
5Gi is the default quota limit in BC Gov OCP provisioned namespaces for `nfs-backup` class storage, teams can request more.  If you are performing backups in a non-prod environment or an environment outside of BC Gov OCP you can use a different class of storage which will have a different default quota associated with it and therefore you may be able to provision more backup storage.

Reply:
I can update this, but the reason this made it into the added section is because it is taken verbatim from the Postgres example.

Reply:
@ashtonmeuser, That would be awesome if you could update both sections.  That's my oversight that this was not caught when the Postgres example was added.